tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/remote/BallerinaRunRemoteDebugTest.java

```diff
@@ -64,25 +67,38 @@ public void testSuspendOnBallerinaFileRun() throws BallerinaTestException {
         String msg = REMOTE_DEBUG_LISTENING + port;
         LogLeecher clientLeecher = new LogLeecher(msg);
         balClient.debugMain("run", new String[]{"--debug", String.valueOf(port),
-            debugTestRunner.testEntryFilePath}, null, new String[]{}, new LogLeecher[]{clientLeecher},
-            debugTestRunner.testProjectPath, 10);
+                        debugTestRunner.testEntryFilePath}, null, new String[]{}, new LogLeecher[]{clientLeecher},
+                debugTestRunner.testProjectPath, 10);
         clientLeecher.waitForText(20000);
     }
 
-    @Test
-    public void testSuspendOnBallerinaJarRun() throws BallerinaTestException {
+    @Test(description = "Tests executable JAR debugging support with [--debug <PORT>] option.")
+    public void testSuspendOnBallerinaJarRun1() throws BallerinaTestException {
+        int port = findFreePort();
+        testBalJarInDebugMode("--debug", String.valueOf(port));
+    }
+
+    @Test(description = "Tests executable JAR debugging support with [--debug=<PORT>] option.")
+    public void testSuspendOnBallerinaJarRun2() throws BallerinaTestException {
+        int port = findFreePort();
+        testBalJarInDebugMode("--debug=" + port);
+    }
+
+    public void testBalJarInDebugMode(String... debugOptions) throws BallerinaTestException {
         String executablePath = Paths.get("target", "bin", testProjectName.replaceAll("-", "_") + ".jar")
-            .toFile().getPath();
+                .toFile().getPath();
         LogLeecher clientLeecher = new LogLeecher(executablePath);
         balClient.runMain("build", new String[]{}, null, new String[]{},
-            new LogLeecher[]{clientLeecher}, debugTestRunner.testProjectPath);
+                new LogLeecher[]{clientLeecher}, debugTestRunner.testProjectPath);
```

Comment:
Formatting issue

Reply:
I just corrected the formatting using wso2 styling guidelines.